src/main/java/QueryServlet.java

```diff
@@ -0,0 +1,68 @@
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import java.util.Map;
+import java.util.HashMap;
+import java.util.stream.Stream;
+import java.util.stream.Collectors;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+
+import com.google.gson.Gson;
+
+/** Servlet that handles census queries from users */
+@WebServlet("/query")
+public class QueryServlet extends HttpServlet {
+
+  // Hardcoded initial choices for which lines of the data table are accessed
+  Map<String, String> lineSelector = 
```

Comment:
Maybe a better name for this? lineSelector does not represent this well enough.

Reply:
Changed to queryToDataRowMapper